dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/fat/src/com/ibm/ws/jaxws/fat/HandlerChainTest.java

```diff
@@ -158,6 +175,26 @@ public void tearDown() throws Exception {
      * }
      */
 
+    /**
+     * Negative test designed to fail unless com.ibm.ws.webservices.ignoreUnknownElements is set to true
```

Comment:
Can you change this comment to: 

```suggestion
     * Negative test designed to fail because optional elements are not supported in incoming responses from a Web Service
```